code/utils.py

```diff
@@ -87,11 +81,8 @@ def get_subfolder_paths(folder: Path) -> List[Path]:
     Returns:
         A list containing the paths of the subfolders.
     """
-    return sorted([
-        folder.joinpath(f.name) for f in folder.iterdir()
-        if ((folder.joinpath(f.name).is_dir()) and (".DS_Store" not in f.name))
-    ],
-                  key=str)
+    return sorted([folder.joinpath(f.name) for f in folder.iterdir() if
+                   ((folder.joinpath(f.name).is_dir()) and (".DS_Store" not in f.name) and (f.suffix.casefold() in IMAGE_EXTS))], key=str)
```

Comment:
This should not check extensions; this function is to list subfolders (see .is_dir()).